backend/sozisel/lib/sozisel/model/event_results.ex

```diff
@@ -47,4 +50,116 @@ defmodule Sozisel.Model.EventResults do
   def change_event_result(%EventResult{} = event_result, attrs \\ %{}) do
     EventResult.create_changeset(event_result, attrs)
   end
+
+  def get_all_event_results(%LaunchedEvent{} = launched_event) do
+    launched_event
+    |> Repo.preload(:event_results)
+    |> then(& &1.event_results)
+  end
+
+  def quiz_summary(%LaunchedEvent{} = event_launched) do
+    quiz_results = get_all_event_results(event_launched)
+
+    total_points =
+      quiz_results
+      |> Enum.flat_map(& &1.result_data.participant_answers)
+      |> Enum.map(& &1.points)
+      |> Enum.sum()
+
+    total_quiz_answer_time =
+      quiz_results |> Enum.map(& &1.result_data.quiz_answer_time) |> Enum.sum()
+
+    participant_answers =
+      quiz_results
+      |> Enum.flat_map(& &1.result_data.participant_answers)
```

Comment:
At work somebody recently pointed me out not to use pipe operator if we have just one :( So in this case just use `Enum.flat_map(quiz_results, ...`